ArchiSteamFarm/Bot.cs

```diff
@@ -540,8 +540,12 @@ internal sealed class Bot : IDisposable {
 							continue;
 						}
 					}
+				} else if (botName.StartsWith("r!", StringComparison.OrdinalIgnoreCase)) {
+					string botPattern = botName.Substring(2);
+					IEnumerable<Bot> regexMatches = Bots.Where(kvp => Regex.Match(kvp.Key, botPattern, RegexOptions.IgnoreCase).Success).Select(kvp => kvp.Value);
+					result.UnionWith(regexMatches);
 				}
-
+				
```

Comment:
😕

Reply:
oh come on. i hate this auto-completion xD